src/components/Carousel/Carousel.js

```diff
@@ -1,7 +1,8 @@
 import React, {useState, useEffect} from 'react';
 import ButtonSlider from 'components/ButtonSlider/ButtonSlider.style';
+import { Link } from 'react-router-dom';
 
-const CarouselComponent = ({className, data}) => {
+const CarouselComponent = ({className, data: { parsedData = [], isLoading }}) => {
```

Comment:
the `isLoading` variable is not being used, are you planing on using it in the future?
```suggestion
const CarouselComponent = ({className, data: { parsedData = [] }}) => {
```

Reply:
I was going for the bonus requirement of presenting a "Loading" pic while the component was loading, but I left it for last and didn't have time to finish it. You can see an example of what I was going for in the ProductDetail component.

Reply:
don't worry. You can still add it in the future.